pkg/sass_api/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
 
 * All uses of classes from the `tuple` package have been replaced by record
   types.
+  
+## 7.2.2
+
+* No user-visible changes.
```

Comment:
Is this still necessary?

Reply:
Yes, it got accidentally deleted in a previous CL but we did release 7.2.2 so it should have a changelog entry.